sdk/src/builder.rs

```diff
@@ -253,6 +301,29 @@ impl Builder {
         Default::default()
     }
 
+    /// Creates a new [`Builder`] for creating a new asset.
+    ///
+    /// # Arguments
+    /// * `source_type` - The type of digital source, such as `DigitalSourceType::Empty` or `DigitalSourceType::TrainedAlgorithmicData`.
+    /// # Returns
+    /// * A new [`Builder`] with the specified source type.
+    /// # Example
+    /// ```rust
+    /// use c2pa::{Builder, DigitalSourceType};
+    /// let builder = Builder::create(DigitalSourceType::Empty);
+    /// ```
+    pub fn create(source_type: DigitalSourceType) -> Self {
+        let mut builder = Self::new();
```

Comment:
Do we want to deprecate `Self::new` in favor of these new constructors?

Reply:
probably, but I'm still sorting this out.  We can always defer when we are ready to do so.